functions/sum/1.0/function.json

```diff
@@ -0,0 +1,60 @@
+{
+  "description": "Sum two numbers with an operator to sym two values into one number (eg, total is 'firstNumber' '+' 'secondNumber')",
+  "label": "Sum",
```

Comment:
I think we should come up with a better name for this function seeing as we are not only doing a `sum`.